src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,25 @@
 import React from 'react';
+import { User } from '../../types/types';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  users: User[];
+  userId: number;
+};
+
+export const UserInfo: React.FC<Props> = ({ users, userId }) => {
+  const user: User | undefined = users.find((
+    currentUser: User,
+  ) => currentUser.id === userId);
+
+  if (!user) {
+    return null;
+  }
+
+  return (
+    user && (
```

Comment:
The `user &&` condition is unnecessary since you already checked if `user` exists in line 17. You can remove it and just return the JSX directly. 🧹